desc/profiles.py

```diff
@@ -811,23 +811,28 @@ class SplineProfile(_Profile):
         - `'catmull-rom'`: C1 cubic centripetal "tension" splines
     name : str
         name of the profile
+    df : array-like
```

Comment:
If I load in a vmec wout file, we assign `SplineProfile` for pressure and iota/current, this then will cause issues as `df` is not assigned and so defaults to `None`, but then there are errors when anything is attempted to be computed from that profile

Reply:
What is the error? `df` is not used unless it is not `None`.